src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DataSourceUtils.java

```diff
@@ -228,13 +229,84 @@ public static boolean isValidDirectory(final Path p) {
      * @param annotationOverridesMap {@link LinkedHashMap} of {@link String}->{@link String} containing any annotation overrides to include in data sources.  Must not be {@code null}.
      * @param transcriptSelectionMode {@link TranscriptSelectionMode} to use when choosing the transcript for detailed reporting.  Must not be {@code null}.
      * @param userTranscriptIdSet {@link Set} of {@link String}s containing transcript IDs of interest to be selected for first.  Must not be {@code null}.
+     * @param funcotatorToolInstance Instance of the {@link Funcotator} {@link GATKTool} into which to add {@link FeatureInput}s.
```

Comment:
Doesn't necessarily have to be `Funcotator`...any tool that is a client of the funcotation engine. Recommend renaming to just `toolInstance`

Reply:
Never null

Reply:
Fixed.